modules/openapi-generator/src/main/resources/nodejs-express-server/config.mustache

```diff
@@ -2,9 +2,9 @@ const path = require('path');
 
 const config = {
   ROOT_DIR: __dirname,
-  URL_PORT: 3000,
-  URL_PATH: 'http://localhost',
-  BASE_VERSION: 'v2',
+  URL_PORT: {{#serverPort}}{{serverPort}}{{/serverPort}}{{^serverPort}}1234{{/serverPort}},
```

Comment:
@kmturley thanks for the PR. Can we keep 3000 instead to make it backward compatible?

Reply:
sure have updated!